client/src/components/MenuToggle.jsx

```diff
@@ -0,0 +1,21 @@
+// Rsuite
+import { IconButton } from 'rsuite';
+import { Icon } from '@rsuite/icons';
+
+// Icons
+import { BiDotsVerticalRounded } from 'react-icons/bi';
+import React from 'react'
+
+const MenuToggle = (props, ref) => {
```

Comment:
We use propType to give the type of props in react, here's an example of how yours should look like

https://legacy.reactjs.org/docs/typechecking-with-proptypes.html